static/js/filter.js

```diff
@@ -99,6 +99,14 @@ function message_matches_search_term(message, operator, operand) {
     case 'sender':
         return people.id_matches_email_operand(message.sender_id, operand);
 
+    case 'huddle-with':
+        var operands_ids = people.pm_with_operand_ids(operand);
+        if (!operands_ids) {
+            return false;
+        }
+
+        return true;
```

Comment:
Well, the real problem here is that I don't think we're actually seeing if the message matches the operand. :)